.github/workflows/integration.yml

```diff
@@ -9,9 +9,13 @@ on:
         description: Otomi installation profile
         default: full
         type: string
+      server_providers:
```

Comment:
How about renaming it to `cloud_provider` ?

Reply:
Good idea, I'll change it.